src/workflows/dags/forecast_prediction_jobs.py

```
@@ -0,0 +1,237 @@
+    @task()
+    def time_related_features(data):
+        return SatelliteMLUtils.time_features(data)
+
+    @task()
+    def lag_features_extraction(data, frequency):
+        return SatelliteMLUtils.lag_features(data, frequency="hourly")
+
+    @task()
+    def make_predictions(data):
+        return ForecastUtils.generate_forecasts(
+            data=data, project_name=project_id, bucket_name=bucket, frequency="hourly"
```

Ruff

Undefined name `project_id`

(F821)